lib/internal/bootstrap/node.js

```diff
@@ -128,6 +129,64 @@
       setupGlobalConsole();
       setupGlobalURL();
       setupGlobalEncoding();
+
+      let errorCodes;
+      let asyncHooks;
+      // declared separately for name, arrow function to prevent construction
+      const queueMicrotask = (callback) => {
+        if (typeof callback !== 'function') {
+          if (errorCodes === undefined) {
+            errorCodes = NativeModule.require('internal/errors').codes;
+          }
+          throw new errorCodes.ERR_INVALID_ARG_TYPE(
+            'callback', 'function', callback);
+        }
+
+        if (asyncHooks === undefined) {
+          asyncHooks = NativeModule.require('internal/async_hooks');
+        }
+
+        const triggerAsyncId = asyncHooks.getDefaultTriggerAsyncId();
+        const asyncId = asyncHooks.newAsyncId();
+        asyncHooks.emitInit(asyncId,
+                            'Microtask',
```

Comment:
The new asyncHooks resource type `Microtask` should be added to doc in https://nodejs.org/dist/latest/docs/api/async_hooks.html#async_hooks_type